src/index.js

```diff
@@ -28,6 +28,10 @@ export const ApolloClientToken: Token<ApolloClient> = createToken(
   'ApolloClientToken'
 );
 
+export const ApolloContextToken: Token<() => mixed | mixed> = createToken(
```

Comment:
Still need to support FusionPlugins, but we can add that inn a follow-up. Thanks for the work here!